extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss

```diff
@@ -91,5 +92,142 @@
 	display: none;
 }
 
+.global-navigation-2016 {
+	.global-nav-search-input-wrapper .autocomplete-container,
+	.notifications-container .notifications-container,
+	.account-navigation-container .user-menu,
+	.AccountNavigation .UserLoginDropdown {
+		margin-top: 1px;
+	}
+
+	.wikia-logo {
+		align-items: flex-end;
+		display: flex;
+		text-decoration: none !important;
+	}
+
+	.wikia-logo__subtitle {
+		border-bottom: 1px solid;
+		border-top: 1px solid;
+		color: #aaa;
+		display: inline-block;
+		font-size: 10px;
+		font-weight: bold;
+		line-height: 16px;
+		margin-left: 8px;
+		text-transform: uppercase;
+	}
+
+	.explore-wikia-dropdown {
+		background-color: $global-wikia-background-color;
+		border: 1px solid rgba(0,0,0,.15);
+		border-top: none;
+		display: none;
+		position: absolute;
+		transition-duration: .5s;
+		transition-property: background-color;
+		z-index: 5000100;
+
+		a {
+			border-top: 1px solid #ddd;
+			color: $global-navigation-text-color;
+			display: block;
+			font-size: 14px;
+			padding: 12px;
+			text-decoration: none;
+			width: 100%;
+
+			&:hover {
+				background-color: $global-navigation-cell-hover-color;
+				text-decoration: none;
+			}
+
+			&.start-wikia {
```

Comment:
yes - it needs to override previous styling. but we could clean this up while removing old (non-2016) nav later